doctrine-mapping.xsd

```diff
@@ -202,7 +202,7 @@
 
   <xs:simpleType name="tablename" id="tablename">
     <xs:restriction base="xs:token">
-      <xs:pattern value="[a-zA-Z_u01-uff.]+" id="tablename.pattern">
+      <xs:pattern value="([^`.][^.]*\.)?([a-zA-Z0-9_$#&#x80;-&#x10FFFF;]+|`.+`)" id="tablename.pattern">
```

Comment:
Note: the proposed part for an unquoted table name, `[a-zA-Z0-9_$#&#x80;-&#x10FFFF;]+`, is (sort of) the _union_ of formats allowed by different vendors (mainly MySQL, PostgreSQL, Oracle).
Would you on the contrary want the _intersection_, e.g. `[a-zA-Z][a-zA-Z0-9_]*` (very restrictive)?

Do you also want that multibyte characters be considered as invalid (like for `fqcn`), even in _quoted_ table names?

And maybe also limit the maximum length? (to e.g. 128 or even 30 chars?)

Reply:
I don't know if we perform binary-unsafe operations on table names in the codebase (specifically: naming strategy, quoting), so being restrictive is OK.

Maximum length is to be avoided, because it is highly platform-dependent, and I've already seen DBs with horrendous generated table names in OracleDB (ascii-only): those would be surely over 64 chars. Unsure if any of the supported platforms goes over 128 chars though.